playground/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest_container
```

Comment:
We chatted about this, I don't have extensive experience with testing "bare" app containers outside of Kube/OCP but I did want to just link for reference e.g. https://github.com/osbuild/bootc-image-builder/tree/main/test which is also using pytest to test a container image, though not using this library.

Reply:
Thanks. I'll check it out.

Reply:
It seems that the container functions were implemented instead of using the plugin.   Personally, I prefer using a plugin.